src/DataLoader/DataLoader.cs

```diff
@@ -307,7 +307,9 @@ private void UpdateState(Func<DataLoaderState, DataLoaderState> update)
 			}
 		}
 
-		/// <inheritdoc />
+		/// <summary>
+		/// Disposes of every strategies and triggers this <see cref="DataLoader"/> had.
```

Comment:
["Every" should be followed by singulars nouns.](https://www.macmillandictionary.com/dictionary/british/every#:~:text=Every%20is%20generally%20used%20before,used%20with%20a%20singular%20verb.)